test/policies/namespace_policy_test.rb

```diff
@@ -8,6 +8,21 @@ def setup
     @policy = NamespacePolicy.new(user: @user)
   end
 
+  test 'named scope with expired memberships' do
+    # assuming personal projects cannot be expired
```

Comment:
This comment can be removed. That is correct that personal projects cannot be expired

Reply:
3a2aeb09e85eb371b5b577b45d56a3ecd01938fc